app/controllers/checkouts_controller.rb

```diff
@@ -9,7 +9,7 @@ class CheckoutsController < ApplicationController
   # GET /checkouts
   # GET /checkouts.json
   def index
-    @checkouts = patron_or_group.checkouts.sort_by { |x| x.sort_key(:due_date) }
+    @checkouts = patron_or_group.checkouts&.sort_by { |x| x.sort_key(:due_date) }
```

Comment:
Is there a reason we have the nil check here?  Doesn't `#checkouts` always return an array? If not, can you make it always return an array?

Reply:
I put this here to address Chris's comment above

Reply:
I don't see the comment you are referring to.

Reply:
Chris asked:
> Does the return value for non-sponsors matter? Or, if it does, could we return an empty array to simplify any downstream consumers?

And his code suggestion didn't include the returned empty array. So I tried to follow that. I can add the array back but this seemed to go on the track of the return value not mattering for non-sponsors.